docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,22 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned for each column, along with a summary row. The following
```

Comment:
along with? Probably in a summary row

Reply:
`SHOW STATS FOR table` returns stats for a single table with one row for each column AND another summary row for the entire table. Maybe `in addition to` would be clearer?

Reply:
(and the "in the column" etc cleaned up)

I think what is really confusing is that only one description says "(``NULL`` for the summary row)" and it really should say that in the descriptions for all column-only values

Reply:
Reworded this table's introduction, and added a "notes" column that describes limitations like whether the value returns on per-column vs table summary rows, and which types are accounted for.

Please have a look and make sure I didn't introduce technical inaccuracies.